frontend/src/views/Landing.tsx

```diff
@@ -1,21 +1,213 @@
+            <Card pad="medium">
+              <CardHeader pad="medium">
+                Cool Project Name
+              </CardHeader>
+              <CardBody pad="medium">
+                This is what the project is about. It is really cool stuff. 
+                Everyone will read every single word of this description
+                because it's so captivating. Wow look at all this information.
+                This project is going so well. I am running out of words to put
+                in order to adequately fill this space. These are more words that
+                I am including because we will probably write a lot more about
+                this project because it's just that cool.
+              </CardBody>
+              <CardFooter style={{ display: "flex", justifyContent: "right", alignItems: "right" }}>
+                <DarkButton>Learn more</DarkButton>
+              </CardFooter>
+            </Card>
+            <Card pad="medium">
+              <CardHeader pad="medium">
+                Cool Project Name
+              </CardHeader>
+              <CardBody pad="medium">
+                This is what the project is about. It is really cool stuff. 
+                Everyone will read every single word of this description
+                because it's so captivating. Wow look at all this information.
+                This project is going so well. I am running out of words to put
+                in order to adequately fill this space. These are more words that
+                I am including because we will probably write a lot more about
+                this project because it's just that cool.
+              </CardBody>
+              <CardFooter style={{ display: "flex", justifyContent: "right", alignItems: "right" }}>
+                <DarkButton>Learn more</DarkButton>
+              </CardFooter>
+            </Card>
+            <Card pad="medium">
+              <CardHeader pad="medium">
+                Cool Project Name
+              </CardHeader>
+              <CardBody pad="medium">
+                This is what the project is about. It is really cool stuff. 
+                Everyone will read every single word of this description
+                because it's so captivating. Wow look at all this information.
+                This project is going so well. I am running out of words to put
+                in order to adequately fill this space. These are more words that
+                I am including because we will probably write a lot more about
+                this project because it's just that cool.
+              </CardBody>
+              <CardFooter style={{ display: "flex", justifyContent: "right", alignItems: "right" }}>
+                <DarkButton>Learn more</DarkButton>
+              </CardFooter>
+            </Card>
+          </Section>
+        </AdditionalMargin>
+        <DarkSection>
+            <Grid gap="medium" columns={{ count: 'fit', size: 'small' }}>

```

Comment:
When using Grids, try to define areas and use them like in the example here: https://v2.grommet.io/grid